target/linux/ar71xx/mikrotik/config-default

```diff
@@ -89,6 +89,7 @@ CONFIG_ATH79_MACH_RBSXTLITE=y
 # CONFIG_ATH79_MACH_TL_WR1041N_V2 is not set
 # CONFIG_ATH79_MACH_TL_WR1043ND is not set
 # CONFIG_ATH79_MACH_TL_WR1043ND_V2 is not set
+# CONFIG_ATH79_MACH_TL_WR1043ND_V4 is not set
```

Comment:
I noticed the 1043_v2 is missing in target/linux/ar71xx/nand/config-default . Is that an error or is there a difference?